test_runtime.sh

```diff
@@ -77,9 +77,9 @@ trap cleanup EXIT
 tar -xf  rootfs.tar.gz -C ${TESTDIR}
 cp runtimetest ${TESTDIR}
 
-ocitools generate --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
+ocitools generate --tty --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
```

Comment:
On 08/05/2016 05:30 PM, keloyang wrote:
> debug info, such as DEBU[0000] validating ...
That is not --tty's function.
If you want info like that, you just need to execute test_runtime.sh with `-l debug`.
--tty is not necessary.